kivy_garden/graph/__init__.py

```diff
@@ -1527,6 +1527,37 @@ def plot_mesh(self, *args):
             vert[k * 8 + 5] = px_ymax
         mesh.vertices = vert
 
+class ScatterPlot(Plot):
+    """
+    ScatterPlot draws using a standard Point object.
+    The pointsize can be controlled by passing `pointsize` when initialising
+    the class. Default pointsize is 1.0.
+
+    >>> plot = ScatterPlot(color=[1, 0, 0, 1], pointsize=5)
+    """
+
+    pointsize = NumericProperty(1)
+
+    def create_drawings(self):
+        from kivy.graphics import Point, RenderContext
+
+        self._grc = RenderContext(
```

Comment:
Please use clearer names. In python it's generally better to write e.g. `self._points_context = RenderContext` and the same for the other objects.

Reply:
And yeah, the line version also uses these cryptics names, but it's better late than never.